### PR TITLE
Add tests for commands

### PR DESCRIPTION
**Pull request by tetafro**
Fixes https://github.com/cristalhq/zookeeper/issues/1

I've added tests and fixed a couple of things in the code.

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ var (
 ^Received: (\d+).*\n^Sent: (\d+).*\n^Connections: (\d+).*\n^Outstanding: (\d+).*
 ^Zxid: (0x[A-Za-z0-9]+).*\n^Mode: (\w+).*\n^Node count: (\d+))`
 
-	consRE = `^ /(?:[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+:(?:[0-9]+))\[[0-9]+\]` +
```
Comment (tetafro) [inline commands.go]: It was a mistake, right? IP address and port were omitted from regexp groups.

File: commands.go [hunk 2]
```diff
@@ -99,14 +99,14 @@ func CallRuok(servers []string, timeout time.Duration) []bool {
 			continue
 		}
 
-		if bytes.Equal(response[:4], iamokResponse) {
```
Comment (tetafro) [inline commands.go]: `allOk` - 5 letters

File: commands.go (hunk 1 above)
Comment (cristaloleg) [reply commands.go]: Yep, they were omitted. Maybe in a separate PR we can catch them and extend `ServerClient` struct, but I'm not sure about that.

File: commands.go (hunk 2 above)
Comment (cristaloleg) [reply commands.go]: Indeed, 5 letters.

**Comment by cristaloleg** [comment]
Thank you @tetafro

File: commands.go [hunk 3]
```diff
@@ -150,8 +150,8 @@ func CallCons(servers []string, timeout time.Duration) ([]*ServerClients, bool)
 				MinLatency:    safeParseInt32(match[12]),
 				AvgLatency:    safeParseInt32(match[13]),
 				MaxLatency:    safeParseInt32(match[14]),
-				Established:   time.Unix(safeParseInt64(match[6]), 0),
-				LastResponse:  time.Unix(safeParseInt64(match[10]), 0),
+				Established:   time.Unix(safeParseInt64(match[6]), 0).UTC(),
+				LastResponse:  time.Unix(safeParseInt64(match[10]), 0).UTC(),
```
[the remark this answers is not in the document]
Comment (tetafro) [reply commands.go]: Forgot to delete.
Fixed.

**Comment by tetafro** [comment]
Also fyi (I'm not a zookeeper expert), here's what I got for `cons` command on my local zookeeper instance:
```
$ telnet 172.20.0.130 2181
...
cons 
 /172.20.0.1:41520[0](queued=0,recved=1,sent=0)
```

Much shorter than your regexp expects. So I just googled example output.

**Comment by cristaloleg** [comment]
Hm, that's interesting, is this ZK 3.5.x ?
There might be difference in output for leader/follower/observer, but I cannot confirm this currently.
I think I'll make an issue to keep it somewhere, thanks for the info.

**Comment by tetafro** [comment]
> Hm, that's interesting, is this ZK 3.5.x ?

Yep, 3.5.5. Just got the latest docker image:
```
docker run -d \
    --name zookeeper \
    --net docknet \
    --ip 172.20.0.130 \
    --env ZOO_4LW_COMMANDS_WHITELIST=* \
    zookeeper:3.5
```

**Pull request merged**